static/app/components/codecov/virtualRenderers/useScrollLeftSync.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import {useRef} from 'react';
+
+// Because of the way that userEvent works, we're unable to use it for these tests, and
+// require the use of fireEvent instead.
+import {fireEvent, render, screen, waitFor} from 'sentry-test/reactTestingLibrary';
+
+import {useScrollLeftSync} from './useScrollLeftSync';
```

Comment:
Out of scope, but wdyt about making this a useScrollSync and add a configurable left/top functionality? There might be other use cases for something like this in Sentry

Reply:
Done!